pytest_spec/patch.py

```diff
@@ -81,7 +81,7 @@ def pytest_runtest_logreport(self, report):
     if not isinstance(word, tuple):
         test_name = _get_test_name(report.nodeid)
         docstring_summary = getattr(report, 'docstring_summary', '')
-        docstring_summary = docstring_summary if docstring_summary else test_name
+        docstring_summary = docstring_summary if docstring_summary else ""
```

Comment:
actually this line can be deleted imho

in my project, i'm using it like this:

```suggestion
        docstring_summary = f" ({docstring_summary})" if docstring_summary else ""
```